clickhouse_sqlalchemy/drivers/http/transport.py

```diff
@@ -46,11 +46,12 @@ def _get_type(type_str):
 
 class RequestsTransport(object):
     def __init__(self, db_url, db_name, username, password, timeout=None,
-                 **kwargs):
+                 verify=None, **kwargs):
```

Comment:
Let's set `verify=True` by default?

Reply:
It makes no difference. If nothing set, it later internally resolves to True.